### PR TITLE
Dev to Master 3.2.13 - Jupiter Epic

### PR DESCRIPTION
**Pull request by schweitzdesign**
Jupiter Epic

### CODE REVIEW

**Comment by schweitzdesign** [comment]
@alexisBimatovs - conflicts

File: .gitignore [hunk 1]
```diff
@@ -14,8 +14,6 @@
 /.bundle
 /vendor/bundle/
 /vendor/ruby/
-/dist
```
Comment (schweitzdesign) [inline .gitignore]: why were these removed from the gitignore?

**Comment by alexisBimatovs** [comment]
ok @schweitzdesign should be cleaned up now, sorry about that!

File: Gemfile.lock [hunk 2]
```diff
@@ -190,4 +190,10 @@ DEPENDENCIES
   sprockets (= 2.12.4)
   sqlite3
   thor
-  
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (schweitzdesign) [inline Gemfile.lock]: Probably didn't want to commit this.

**Comment by alexisBimatovs** [comment]
@schweitzdesign back to you, sorry about that commit 🦅

**Comment by schweitzdesign** [comment]
Looks good to me!
